migrations/frontend/1665056530_autogeneratewebhookid/up.sql

```
@@ -0,0 +1,4 @@
+ALTER TABLE IF EXISTS webhooks ALTER COLUMN id SET DEFAULT gen_random_uuid();
```

You probably don't want the `IF EXISTS` here and silently ignore the webhooks table missing completely.

---

@efritz why should we do it? I just don't get it

---

It shows a definite failure sooner. The way it's written now:

- will have the same effect if ran twice (idempotent)
- requires that the webhooks table actually exists

The way it was written before:

- won't error if there's no webhooks table (hides an error condition)
- **will** error if the column exists and has a default already (not idemptent)

---

You *do* what `DROP TABLE IF EXISTS` on the down direction, though. Otherwise you'll crash when it doesn't exist (but that's the goal of `down.sql`).